### DISCUSSION TITLE
How to use prisma with deno?

### DISCUSSION THREAD

**Discussion by Amatewasu** [prisma/prisma#25537] [Q&A]
### Question

I would like to use Prisma with deno without using Prisma accelerate. The only source I found to use Deno with Prisma uses accelerate.

By following the offical deno tutorial ([How to create a RESTful API with Prisma and Oak](https://docs.deno.com/runtime/tutorials/how_to_with_npm/prisma/)), they suggest using importing the generated client using the following import:
`import { Prisma, PrismaClient } from "../generated/client/deno/edge.ts";`

But thereby I have the following error:
```
error: Uncaught (in promise) PrismaClientKnownRequestError: 
Invalid `prisma.project.create()` invocation:


Error validating datasource `db`: the URL must start with the protocol `prisma://`
```

It looks like it is because my database url begins with `postgresql://`, which is normal if i do not use any pool.

Do you have any idea?

**Comment by Amatewasu** [comment]
With this workaround, it works great: https://github.com/prisma/prisma/issues/2452#issuecomment-2452980392

**Reply by ludralph** [reply]
Thanks for sharing the workaround. It would be helpful to others as well.
